app/qml/map/MapCanvas.qml

```diff
@@ -8,110 +8,50 @@
  ***************************************************************************/
 
 /*
- * The source code forked from https://github.com/qgis/QGIS on 25th Nov 2022
+ * Previous implementation was forked from https://github.com/qgis/QGIS on 25th Nov 2022
```

Comment:
I would argue that if this implementation is complete revamp, we can remove this notice ( or you can name the file MapCanvasV2 and remove the original file )